src/dpp/guild.cpp

```diff
@@ -29,7 +29,7 @@
 
 using json = nlohmann::json;
 
-const std::map<std::string, dpp::guild_flags> featuremap = {
+const std::map<std::string, std::variant<dpp::guild_flags, dpp::guild_flags_extra>> featuremap = {
```

Comment:
does this work? is it only working on the guild_flags being uint32_t and guild_flags_extra being uint8_t right now?